index.js

```diff
@@ -44,7 +44,7 @@ function YahooFantasy(consumerKey, consumerSecret) {
     team: teamResource(),
     teams: teamsCollection(),
     transaction: transactionResource(),
-    // transactions: transactionsCollection(),
+    transactions: transactionsCollection(),
```

Comment:
I've been doing a lot of refactoring... there's a good chance I'll have an update by the weekend... would change a lot of this :S